src/main.rs

```diff
@@ -96,7 +96,20 @@ fn main() {
                 }
             }
         }
-
+        if debugger.reset_triggered {
+            debugger.reset_triggered = false;
+            let new_game: Option<Game> = opts.rom.as_ref().and_then(|romname| {
+                Game::new(romname.clone(), context.joypad.clone(), true).map_or_else(
+                    |e| {
+                        log::error!("while creating game context for {}: {:?}", romname, e);
+                        None
+                    },
+                    Option::Some,
+                )
+            });
+            drop(game);
```

Comment:
`drop` isn't required here, it should be called when assigning `game` to `new_game`